<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="1266" started-at="2017-11-16T15:34:49Z" finished-at="2017-11-16T15:34:51Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="1266" started-at="2017-11-16T15:34:49Z" finished-at="2017-11-16T15:34:51Z">
      <class name="nh.ui.antomation.test.cases.DemoTest">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:nh.ui.antomation.test.cases.DemoTest@7085bdee]" name="beforeSuite" is-config="true" duration-ms="5048" started-at="2017-11-16T23:34:44Z" finished-at="2017-11-16T23:34:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="indexPage()[pri:0, instance:nh.ui.antomation.test.cases.DemoTest@7085bdee]" name="indexPage" is-config="true" duration-ms="747" started-at="2017-11-16T23:34:49Z" finished-at="2017-11-16T23:34:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- indexPage -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:nh.ui.antomation.test.cases.DemoTest@7085bdee]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2017-11-16T23:34:50Z" finished-at="2017-11-16T23:34:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void nh.ui.antomation.test.cases.DemoTest.testChromeDriver()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="testChromeDriver()[pri:0, instance:nh.ui.antomation.test.cases.DemoTest@7085bdee]" name="testChromeDriver" duration-ms="185" started-at="2017-11-16T23:34:50Z" finished-at="2017-11-16T23:34:50Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[没有找到测试对象,对象名称[bdSearchInput]!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 没有找到测试对象,对象名称[bdSearchInput]!
	at org.testng.Assert.fail(Assert.java:94)
	at nh.ui.automation.tools.common.ByType.getByType(ByType.java:56)
	at nh.ui.automation.tools.common.Action.objInfo(Action.java:97)
	at nh.ui.automation.tools.common.ElementCommon.getElement(ElementCommon.java:36)
	at nh.ui.automation.tools.common.Action.sendkey(Action.java:126)
	at nh.ui.antomation.test.cases.DemoTest.testChromeDriver(DemoTest.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testChromeDriver -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:nh.ui.antomation.test.cases.DemoTest@7085bdee]" name="afterMethod" is-config="true" duration-ms="310" started-at="2017-11-16T23:34:50Z" finished-at="2017-11-16T23:34:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testChromeDriver status=FAILURE method=DemoTest.testChromeDriver()[pri:0, instance:nh.ui.antomation.test.cases.DemoTest@7085bdee] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- nh.ui.antomation.test.cases.DemoTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
